# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/acerhk/no_depmod.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -pruN acerhk-0.5.27-orig/Makefile acerhk-0.5.27/Makefile
--- acerhk-0.5.27-orig/Makefile	2005-09-09 14:59:26.000000000 +0200
+++ acerhk-0.5.27/Makefile	2005-09-09 14:59:39.000000000 +0200
@@ -63,4 +63,3 @@ install: $(TARGET)
 	#cp -v $(TARGET) /lib/modules/$(KERNELVERSION)/kernel/drivers/char
 	mkdir -p /lib/modules/$(KERNELVERSION)/extra
 	cp -v $(TARGET) /lib/modules/$(KERNELVERSION)/extra/
-	depmod -a
